bitsail-connectors/connector-ftp/src/test/resources/data/csv/upload2/test1.csv

```diff
@@ -0,0 +1,2 @@
+c0,c1,c2,c3,c4,c5,c6,c7,c8
+222,bbb,-9.1,1.12345,true,2022-11-01,999999999,1669705101,123456.789
```

Comment:
I'd like to see how read line function, can we add more data into the test file？

Reply:
I added 5 records to each csv file, you can use the csv file for testing. By the way, the `SftpSourceITCase` used these csv file.